style.css

```diff
@@ -1,8 +1,176 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
+* {
+    margin: 0;
 }
+
```

Comment:
- remove empty lines
- the order of css rules should be first parent, then its childs/states like hover. For eg. `.card` should come first, then `.card:hover` and `.card image` as such.
- You can use css variables for values like color which is used again and again.